README.md

```diff
@@ -1,17 +1,109 @@
 # Mira S3 Webhosting Sample Application
 
-This application is a very basic application, that provides ready to use project setup for [Mira accelerator].
+This application is a very basic application, that provides a ready to use project setup for the [Mira accelerator].
```

Comment:
ready-to-use????  I think grammatically you don't hyphenate after a word ending with 'y', however.

Reply:
It's before the noun, so should be hyphenated...will amend